### PR TITLE
Update Makefile

### PR DESCRIPTION
**Pull request by sirouk**
prepped and ready

### CODE REVIEW

File: tools/rescue/Makefile [hunk 1]
```diff
@@ -23,41 +19,66 @@ ifndef DB_PATH
 DB_PATH=${DATA_PATH}/db
 endif
 
+
+ifndef VALS_FILE
+VALS_FILE=~/libra-framework/tools/rescue/fixtures/fork/validators.json
```
Comment (0o-de-lally) [inline tools/rescue/Makefile]: Can you make this into a path using variables please

File: tools/rescue/Makefile [hunk 2]
```diff
@@ -73,15 +94,17 @@ backup-db:
 		echo "Backup successful: $$backup_dir"; \
 	fi
 
+fetch-db:
+	@echo "This will be replaced by either starting a fullnode or using a snapshot from the epoch-archive-mainnet repo."
+	@echo "For simplicity, we will use the db from the machine that maintains the epoch-archive-mainnet repo."
+	
+	@echo "Fetching database..."
+	rm -rf ${DB_PATH}
+	cd ${DATA_PATH}/ && \
```
Comment (0o-de-lally) [inline tools/rescue/Makefile]: use Makefile working directory instead of `cd` and inline commands

**Review by 0o-de-lally — CHANGES_REQUESTED** [review]

File: tools/rescue/Makefile (hunk 1 above)
Comment (sirouk) [reply tools/rescue/Makefile]: Done, needs review.

File: tools/rescue/Makefile (hunk 2 above)
Comment (sirouk) [reply tools/rescue/Makefile]: Done, needs review.

**Pull request merged**